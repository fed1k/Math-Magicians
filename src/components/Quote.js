import React from "react";
const Quote = () => {
    return (
        <div className="quote-page-div">
            <p className="p">Mathematics is not about numbers, equations, computations or algorithms: it is about understanding. - William Paul Thurston</p>
        </div>
    );
}
 
export default Quote;