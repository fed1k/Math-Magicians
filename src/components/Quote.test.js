import React from "react";
import Quote from "./Quote";
import renderer from 'react-test-renderer';

it('should render correctly', ()=>{
    const tree = renderer
        .create(<Quote />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})
