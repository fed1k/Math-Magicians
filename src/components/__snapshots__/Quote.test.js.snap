// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="quote-page-div"
>
  <p
    className="p"
  >
    Mathematics is not about numbers, equations, computations or algorithms: it is about understanding. - William Paul Thurston
  </p>
</div>
`;
